source/src/molecular-unfolding/cdk/utils/custom-resource-lambda/create-event-rule/index.ts

```diff
@@ -0,0 +1,138 @@
+const {
+    CloudFormationClient,
+    CreateStackCommand,
+    UpdateStackCommand,
+    DeleteStackCommand,
+    DescribeStacksCommand
+} = require("@aws-sdk/client-cloudformation");
+
+const {
+    readFileSync
+} = require('fs');
+
+import {
+    Context
+} from './handler';
+
+import {
+    CloudFormationCustomResourceEvent,
+} from './cloudformation-custom-resource';
+
+
+exports.handler = async function (event: CloudFormationCustomResourceEvent, context: Context) {
+    try {
+        await _handler(event, context)
+        console.log("=== complete ===")
+    } catch (e: any) {
+        console.error(e.message);
+        throw e
+    }
+}
+
+function sleep(millis: number) {
+    return new Promise(resolve => setTimeout(resolve, millis));
+}
+
+async function _handler(event: CloudFormationCustomResourceEvent, context: Context) {
+    const currentRegion = process.env.AWS_REGION
+    let RequestType = event.RequestType
+
+    console.log("functionName: " + context.functionName)
+    console.log("currentRegion: " + currentRegion)
+    console.log("RequestType: " + RequestType)
+
+    if (currentRegion == "us-west-2") {
+        console.log('ignore creating event rule')
+        return
+    }
+    const config = {
+        region: "us-west-2"
+    }
+
+    const cf_client = new CloudFormationClient(config);
+    let templateBody = readFileSync(`${__dirname}/template.json`, 'utf-8')
+    const stackName = `QRSFDD-BraketEventTo${currentRegion}`
```

Comment:
if stack name is fixed, how about deploying the solution multiple times?

Reply:
it will update the it. even deploying multiple times, we only need one bridge configed.

Reply:
if sharing the same event bridge rule, how about handling with the deleting stack when deploying multiple times?

Reply:
if it could be done in this PR, create new issue to track it.